### PR TITLE
Add support for nginx proxy to docker-compose

### PR DESCRIPTION
**Pull request by planetf1**
* Adds nginx proxy to docker-compose 
* Polymer UI should now be accessed via https://localhost:10443 in a docker-compose environment
* Docs to be updated in separate PR

### CODE REVIEW

File: open-metadata-resources/open-metadata-deployment/compose/tutorials/.env [hunk 1]
```diff
@@ -6,6 +6,10 @@ egeria_version=2.8-SNAPSHOT
 #
 # Version of egeria-react-ui to use
 egeria_react_ui_version=2.7.0-rc.0
+
+# Version of egeria-ui to use
```
Comment (sarbull) [inline open-metadata-resources/open-metadata-deployment/compose/tutorials/.env:10]: 👍

**Review by sarbull — APPROVED** [review]

**Review by lpalashevski — APPROVED** [review]

**Comment by lpalashevski** [comment]
Looks good and I already tested the similar setup locally usign the same egeria-ui docker image and local nginx configuration that is almost identical with this.

File: open-metadata-resources/open-metadata-deployment/compose/tutorials/.env (hunk 1 above)
Comment (planetf1) [reply open-metadata-resources/open-metadata-deployment/compose/tutorials/.env:10]: I've set to latest for now.. this needs some refinement moving forwards. for the next release we can consider setting it to a fixed value after the branch - open to whatever you are comfortable with.

**Comment by planetf1** [comment]
> Looks good and I already tested the similar setup locally usign the same egeria-ui docker image and local nginx configuration that is almost identical with this.

Super.  I'm think we should discuss deprecating the compose environment moving forward, and focus on k8s as it's more flexible. to mitigate the k8s knowledge needed I'll improve/add docs around using microk8s or kind, which works well in a constrained environment and can run in docker

**Pull request merged**